CONTRIBUTING.md

```diff
@@ -36,13 +51,6 @@ The following `make` target is run on CI to verify the project structure:
 make ensure-generate-is-noop
 ```
 
-### Pre-requisites
-* Install [Go](https://golang.org/doc/install).
-* Install [Kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/).
```

Comment:
is kustomize no longer needed?

Reply:
It is, but we install it locally as needed, so the user doesn't need to do anything.